src/forms/Checkbox.jsx

```diff
@@ -102,6 +101,7 @@ export class Checkbox extends React.PureComponent {
 					onChange={this.onChange}
 					readOnly={!this.onChange || disabled}
 					name={name}
+					aria-label={`${label} - checkbox ` || 'checkbox'}
```

Comment:
Left side will be always true, because it is not empty string.
If label is undefined, it will be "undefined - checkbox".
![Screenshot from 2023-10-06 14-20-06](https://github.com/meetup/meetup-web-components/assets/23707157/98d70f24-1f8b-4efc-9bb6-34b4fbc0251f)

Reply:
yep label also can be a element so I switch it to
 aria-label={ typeof label === 'string' ? \`${label} - checkbox \` : 'checkbox' }